### DISCUSSION TITLE
Navigation stuck on initialising 11% after Carplay activation.

### DISCUSSION THREAD

**Discussion by Landy989** [Mr-MIBonk/M.I.B._More-Incredible-Bash#523] [General]
I have no idea what has caused this as I have done this procedure before with success... I tried to update maps to the newest version to see if it would budge but I just get NavDB_Check timeout. Am I able to restore from backup? if so, a quick how to or any help at all would be really appreciated.

MHI2_ER_AU43X_P5151
Software:1339

My zipped backup is too big to attach, so if there are any supported file sharing options I am happy to upload to there.

**Comment by Landy989** [comment]
[MU1339-JTH-07619.06.193404008D.log](https://github.com/user-attachments/files/16475343/MU1339-JTH-07619.06.193404008D.log)

**Comment by Landy989** [comment]
Just for anyone else who may need this help or information in the future I was able to fix this. It may have been a complete fluke but it may work for you.

Download and install the toolbox and go to: customisation -> coding and on byte 0 the brand was blank. I changed it to Audi and it immediately threw me out of toolbox and showed me the nav screen at 100% and my maps came back. it could be worth a try.

**Reply by williamsrh** [reply]
Hello Landy, What do you mean by download and install the toolbox? I'm having a similar issue so this might be really helpful.
